eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState/IncrementalStake.hs

```diff
@@ -113,34 +109,35 @@ addCompact Invert (CompactCoin n) (CompactCoin m) = CompactCoin (m - fromIntegra
 --   This function has a non-incremental analog 'aggregateUtxoCoinByCredential' . In this incremental
 --   version we expect the size of the UTxO to be fairly small. I.e the number of inputs and outputs
 --   in a transaction, which is aways < 4096, not millions, and very often < 10).
-incrementalAggregateUtxoCoinByCredential ::
```

Comment:
We should maybe update the haddock above where it  says ` For inserts (the mode is Id) and for deletes (the mode is Invert).`   (maybe in a haddock for the new function parameter?)

Reply:
Good point, documentation got outdated. I've change `Id` to `+` and `Invert` to `-` in the haddock.
There is really no need to document it into a huge amount of detail, since that is an internal function and it is pretty clear what it does when one looks at the code.

Reply:
I agree